itest/send_test.go

```diff
@@ -13,6 +16,50 @@ import (
 // testBasicSend tests that we can properly send assets back and forth between
 // nodes.
 func testBasicSend(t *harnessTest) {
+	var (
+		ctxb = context.Background()
+		wg   = sync.WaitGroup{}
+	)
+
+	// Subscribe to receive assent send events from primary taro node.
+	eventNtfns, err := t.tarod.SubscribeSendAssetEventNtfns(
+		ctxb, &tarorpc.SubscribeSendAssetEventNtfnsRequest{},
+	)
+	require.NoError(t.t, err)
+
+	// Test to ensure that we execute the transaction broadcast state.
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+
+		foundSendSates := make(map[string]struct{})
+		for {
+			event, err := eventNtfns.Recv()
+
+			// Break if we get an EOF, which means the stream was
+			// closed.
+			if err != nil &&
+				strings.Contains(err.Error(), io.EOF.Error()) {
+
+				break
+			}
+
+			// If err is not EOF, then we expect it to be nil.
+			require.NoError(t.t, err)
+
+			// Check for the transaction broadcast state.
+			switch eventTyped := event.Event.(type) {
+			case *tarorpc.SendAssetEvent_ExecuteSendStateEvent:
+				executeSendStateEvent := eventTyped.ExecuteSendStateEvent
+				foundSendSates[executeSendStateEvent.SendState] = struct{}{}
+			}
+		}
+
+		// Ensure that we have found the broadcast state.
+		_, ok := foundSendSates["SendStateBroadcast"]
```

Comment:
See comment about the enum here, then we can just check this directly and not force callers to remember what the string of `SendStateBroadcast` is.